src/main/java/teammates/common/datatransfer/questions/FeedbackRankRecipientsQuestionDetails.java

```diff
@@ -189,6 +189,7 @@ public String getQuestionResultStatisticsHtml(
             templateToUse = FormTemplates.RANK_RESULT_RECIPIENT_STATS;
         }
         DecimalFormat df = new DecimalFormat("#.##");
+        Map<String, Integer> recipientSelfRanks = generateSelfRankForEachRecipient(responses);
```

Comment:
@leeyimin , in that case, the self-rank column can be displayed in CSV statistics irrespective of `shouldSelfBeExcludedFromRankings(question)`, am I right?